src/D2L.Bmx/WriteHandler.cs

```diff
@@ -85,6 +85,27 @@ bool useCredentialProcess
 				+ $" --role {awsCredsInfo.Role}"
 				+ $" --duration {awsCredsInfo.Duration}";
 		} else {
+			if( File.Exists( SharedCredentialsFile.DefaultConfigFilePath ) ) {
+				string profileName = $"profile {profile}";
```

Comment:
nit: it's really the section name rather than profile name
```suggestion
				string sectionName = $"profile {profile}";
```

Reply:
I feel like there's an opportunity to refactor and dedupe some logic and declarations in this whole "write" area. It's getting a bit messy now. But we can do it another time.

Reply:
Specifically the section removing part or something else?

Reply:
everything about handling AWS creds and config files